MPF.Core/UI/ViewModels/MainViewModel.cs

```diff
@@ -160,6 +161,35 @@ public string OutputPath
         }
         private string _outputPath;
 
+        /// <summary>
+        /// Currently provided output path, with < > variables evaluated
+        /// Not guaranteed to be a valid path
+        /// </summary>
+        public string OutputPathEvaluated
+        {
+            get
+            {
+                string system_long = this._currentSystem.LongName() ?? "Unknown System";
+                string system_short = this._currentSystem.ShortName() ?? "unknown";
+                string media_long = this._currentMediaType.LongName() ?? "Unknown Media";
+                string program = this._currentProgram.ToString() ?? "Unknown Program";
+                string program_short = program == "DiscImageCreator" ? "DIC" : program;
+                string label = this._currentDrive?.VolumeLabel ?? "track";
+                string date = DateTime.Today.ToString("yyyyMMdd");
+                string datetime = DateTime.Now.ToString("yyyyMMdd-HHmmss");
+
+                return _outputPath
```

Comment:
If you can find a reasonable way to add this as hovertext for either the main UI or the default output path in the settings window, that would be appreciated.

Reply:
I will add it to the options window only, as the main window currently has no hover text. Also, variables for the default path will be the main use of this. Also also, users that find it in the options window will figure out they can use it in the main window too.